README.md

```diff
@@ -3,10 +3,56 @@
 Tools for automatically debugging dependent tests.
 Currently a work in progress.
 
+## Quickstart
+
+### Use Pre-Compiled Script
```

Comment:
Once the script is moved to the scripts directory, be sure to change these instructions to reflect that. It would also be good to explain what this script actually does and to give it a more descriptive name.